client/src/components/FloatingLabel.js

```diff
@@ -0,0 +1,19 @@
+import styled from "@emotion/styled";
+
+export const FloatingLabel = styled.label`
```

Comment:
awesome work 🎉  it looks really neat and has a nice animation 👍 🚀

Reply:
I agree, looks awesome! 🐦